src/styles/getModalStyles.js

```diff
@@ -131,6 +133,32 @@ export default (type, windowDimensions, popoverAnchorPosition = {}) => {
             animationIn = 'fadeIn';
             animationOut = 'fadeOut';
             break;
+        case CONST.MODAL.MODAL_TYPE.CONTEXT_MENU:
```

Comment:
As noted above, I don't think that this modal type is necessary. But if we _did_ keep it, then we could simplify this and remove a lot of duplicate code:

```javascript
case CONST.MODAL.MODAL_TYPE.POPOVER:
case CONST.MODAL.MODAL_TYPE.CONTEXT_MENU:
    // all the default popover stuff
    
    if (type === CONST.MODAL.MODAL_TYPE.CONTEXT_MENU) {
        // This is the only difference between POPOVER and CONTEXT_MENU
        coverScreen = false;
        hasBackdrop = false;
    }
    break;
```

Reply:
Makes a lot of sense, I was just not able to think to minimize it but I knew It can be.